### PR TITLE
Adds shallow water mooring ingestion csvs

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP11SOSM/CP11SOSM_D00001_ingest.csv
Comment (reedan88) [inline CP11SOSM/CP11SOSM_D00001_ingest.csv:1]: Looks good.

File: CP12CNSW/CP12CNSW_D00001_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#,,CP12CNSW-MFI01-02-ADCPTQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+#,,"CP12CNSW-MFI01-02-CTDMOS031	",telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
```
Comment (reedan88) [inline CP12CNSW/CP12CNSW_D00001_ingest.csv:3]: I think you pasted in some extra quotation marks

File: CP12CNSW/CP12CNSW_D00001_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#,,CP12CNSW-MFI01-02-ADCPTQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+#,,"CP12CNSW-MFI01-02-CTDMOS031	",telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+# ,,CP12CNSW-MFI01-02-WAVSSQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+#,,CP12CNSW-SBI01-02-CTDMOS011,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP12CNSW/D00001/irid2shore/stc_status*.txt,CP12CNSW-SBS01-00-STCENG000,telemetered,Available,
+#,,CP12CNSW-SBS01-03-VELPTA000,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP12CNSW/D00001/3dmgx3/*3dmgx3.log,"CP12CNSW-SBS01-05-MOPAK0000	",telemetered,Available,
```
Comment (reedan88) [inline CP12CNSW/CP12CNSW_D00001_ingest.csv:8]: Have quotation marks and white space in the reference designator

File: CP12WESW/CP12WESW_D00001_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,12 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#,,CP12WESW-MFI01-02-ADCPTQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+#,,"CP12WESW-MFI01-02-CTDMOS031	",telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
```
Comment (reedan88) [inline CP12WESW/CP12WESW_D00001_ingest.csv:3]: Extra quotation marks around the reference designator

File: CP12WESW/CP12WESW_D00001_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,12 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#,,CP12WESW-MFI01-02-ADCPTQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+#,,"CP12WESW-MFI01-02-CTDMOS031	",telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+# ,,CP12WESW-MFI01-02-WAVSSQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+#,,CP12WESW-SBI01-02-CTDMOS011,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP12WESW/D00001/irid2shore/stc_status*.txt,CP12WESW-SBS01-00-STCENG000,telemetered,Available,
+#,,CP12WESW-SBS01-03-VELPTA000,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment 
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP12WESW/D00001/3dmgx3/*3dmgx3.log,"CP12WESW-SBS01-05-MOPAK0000	",telemetered,Available,
```
Comment (reedan88) [inline CP12WESW/CP12WESW_D00001_ingest.csv:8]: Extra quotation marks and white space in the reference designator

**Review by reedan88 — CHANGES_REQUESTED** [review]
Some of the reference designators have quotation marks and white space that should probably be removed (even if they aren't active)

File: CP12CNSW/CP12CNSW_D00001_ingest.csv
Comment (reedan88) [inline CP12CNSW/CP12CNSW_D00001_ingest.csv:1]: Looks good.

File: CP11SOSM/CP11SOSM_D00001_ingest.csv
Comment (reedan88) [inline CP11SOSM/CP11SOSM_D00001_ingest.csv:1]: Looks good.

File: CP12WESW/CP12WESW_D00001_ingest.csv
Comment (reedan88) [inline CP12WESW/CP12WESW_D00001_ingest.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]

File: CP11SOSM/CP11SOSM_D00001_ingest.csv [hunk 5]
```diff
@@ -8,7 +8,7 @@ mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_telemetered_driver,/omc_data/whoi/OM
 mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl36/pco2w/*.pco2w.log,CP11SOSM-MFD36-05-PCO2WB000,telemetered,Available,
 mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl36/phsen2/*.phsen2.log,CP11SOSM-MFD36-06-PHSEND000,telemetered,Available,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl37/syslog/*.syslog.log,CP11SOSM-MFD37-00-DCLENG000,telemetered,Available,
-mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl37/optaa2_decimated/*.optaa2.log,CP11SOSM-MFD37-01-OPTAAD000,telemetered,Available,
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl37/optaa2_decimated/*.optaa2.log,CP11SOSM-MFD37-01-OPTAAD000,telemetered,Not Expected,Failed on deployment
```
Comment (snwhite) [inline CP11SOSM/CP11SOSM_D00001_ingest.csv:11]: Looks good.

File: CP12WESW/CP12WESW_D00001_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+# ,,CP12WESW-MFI01-02-WAVSSQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
+#,,CP12WESW-SBI01-02-CTDMOS011,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP12WESW/D00001/irid2shore/stc_status*.txt,CP12WESW-SBS01-00-STCENG000,telemetered,Available,
+#,,CP12WESW-SBS01-03-VELPTA000,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP12WESW/D00001/3dmgx3/*3dmgx3.log,CP12WESW-SBS01-05-MOPAK0000,telemetered,Available,
+#,/omc_data/whoi/OMC/CP12WESW/D00001/imm/prkt/*.DAT,CP12WESW-WFP01-00-WFPENG001,telemetered,Expected,Parser tbd
+#,/omc_data/whoi/OMC/CP12WESW/D00001/imm/prkt/*.DAT,CP12WESW-WFP01-01-CTDPFP000,telemetered,Expected,Parser tbd
+#,/omc_data/whoi/OMC/CP12WESW/D00001/imm/prkt/*.DAT,CP12WESW-WFP01-02-DOFSTP000,telemetered,Expected,Parser tbd
+#,/omc_data/whoi/OMC/CP12WESW/D00001/imm/prkt/*.DAT,CP12WESW-WFP01-03-FLORTP000,telemetered,Expected,Parser tbd
```
Comment (snwhite) [inline CP12WESW/CP12WESW_D00001_ingest.csv:12]: Looks fine.  Like 4 has an extra space after the #.

File: CP12CNSW/CP12CNSW_D00001_ingest.csv [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+# ,,CP12CNSW-MFI01-02-WAVSSQ010,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
+#,,CP12CNSW-SBI01-02-CTDMOS011,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP12CNSW/D00001/irid2shore/stc_status*.txt,CP12CNSW-SBS01-00-STCENG000,telemetered,Available,
+#,,CP12CNSW-SBS01-03-VELPTA000,telemetered,Not Expected,No telem data; not hooked up inductively for this deployment
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP12CNSW/D00001/3dmgx3/*3dmgx3.log,CP12CNSW-SBS01-05-MOPAK0000,telemetered,Available,
+#,/omc_data/whoi/OMC/CP12CNSW/D00001/imm/prkt/*.DAT,CP12CNSW-WFP01-00-WFPENG001,telemetered,Expected,Parser tbd
+#,/omc_data/whoi/OMC/CP12CNSW/D00001/imm/prkt/*.DAT,CP12CNSW-WFP01-01-CTDPFP000,telemetered,Expected,Parser tbd
+#,/omc_data/whoi/OMC/CP12CNSW/D00001/imm/prkt/*.DAT,CP12CNSW-WFP01-02-DOFSTP000,telemetered,Expected,Parser tbd
+#,/omc_data/whoi/OMC/CP12CNSW/D00001/imm/prkt/*.DAT,CP12CNSW-WFP01-03-FLORTP000,telemetered,Expected,Parser tbd
```
Comment (snwhite) [inline CP12CNSW/CP12CNSW_D00001_ingest.csv:12]: Looks fine.  Like 4 has an extra space after the #.

**Review by snwhite — APPROVED** [review]

**Pull request merged**